### ISSUE TITLE
[Edit and update]: Add the ability for teams to use text + icon button for the editable card

### ISSUE THREAD

**Issue by Ashley-Johnson28** [carbon-design-system/ibm-products#5581]
### The problem

Currently teams are limited to using an icon to trigger the '[editable cards'](https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-edit-and-update-editupdatecards--edit-update-cards&globals=viewport:basic). This is very light weight and easily missed. Our team would love to have the ability to have the word 'edit' by the icon.

[Slack convo](https://ibm-casdesign.slack.com/archives/CQGR0HC05/p1719238124549539)

### The solution

Use a text + icon ghost button for the editable cards like the design documentation indicates. 

### Design link

https://pages.github.ibm.com/cdai-design/pal/patterns/edit/usage/#cards

<img width="950" alt="Screenshot 2024-06-24 at 2 52 30 PM" src="https://github.com/carbon-design-system/ibm-products/assets/51169316/0649c274-f47f-401c-a89b-09409fda0dd2">

### Owner/main maintainer(s)

Carbon for products

### Product/offering

NS1 Connect

### Business priority

Low Priority = release date is not dependent on fix or not upcoming

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate issues

**Comment by Ashley-Johnson28** [comment]
> Hi @Ashley-Johnson28, is this something your team can override currently?

Let me reach out to my devs. Will let you know.

**Comment by ljcarot** [comment]
@Ashley-Johnson28 just checking if you've been in touch with dev regarding ability to override so you are not blocked by this.
